include/picongpu/CMakeLists.txt

```diff
@@ -384,7 +386,7 @@ target_link_libraries(picongpu-hostonly PUBLIC ${LIBS})
 
 
 cupla_add_executable(picongpu
-    ${SRCFILES}
+    "main.cpp"
```

Comment:
`${CMAKE_CURRENT_SOURCE_DIR}/main.cpp` ?

Reply:
Yes must be the absolute pass, I missed to add my last change^^